### ISSUE TITLE
I want to know when the fixed bug is updated to this library

### ISSUE THREAD

**Issue by LJYq** [GoogleChrome/android-browser-helper#47]
I want to know when the fixed bug is updated to this library.
implementation 'com.google.androidbrowserhelper:androidbrowserhelper:0.1.0-alpha1'

**Comment by PEConn** [comment]
Sorry, could you please link the bug you're talking about?

**Comment by andreban** [comment]
We have released version 1.0.0 of the library, which can be used with: 

```
dependencies {
    //...
    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:1.0.0'
}
```

If you find any outstanding issue, please, file a bug and we'll do our best to fix it.
